datascroller/scroller.py

```diff
@@ -3,19 +3,27 @@
 import curses
```

Comment:
Can we stick with pep 8 import order:

Standard library imports.
Related third party imports.
Local application/library specific imports.

https://www.python.org/dev/peps/pep-0008/